ListingManager.Tests/ListingManager.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFrameworks>netcoreapp3.1; net7.0</TargetFrameworks>
```

Comment:
Any reason for multi-targeting rather than just upgrade?

Reply:
Nope, should I not do this in this case

Reply:
For apps (not libraries), unless you have a reason to target multiple TFMs I typically just upgrade them. Enables more features with conditionals and generally feels easier IMO.